ale/drivers/base.py

```diff
@@ -439,21 +697,42 @@ def sun_velocity(self):
         return [sun_state[3:6].tolist()]
 
     @property
-    def sensor_position(self):
-
-        if not hasattr(self, '_sensor_position'):
-            self._compute_ephemerides()
-        return self._sensor_position.tolist()
+    def _sensor_position(self):
+        if not hasattr(self, '_position'):
+            eph = []
+            current_et = self.starting_ephemeris_time
+            for i in range(self.number_of_ephemerides):
+                state, _ = spice.spkezr(self.spacecraft_name,
+                                        current_et,
+                                        self.reference_frame,
+                                        'NONE',
+                                        self.target_name,) # If this is the sensor, insufficient, if this is the spacecraft, it works? Huh?
```

Comment:
The SPKs store the position of the whole spacecraft, not the individual sensors. So, spacecraft instead of sensor is correct.

Reply:
That's not my comment, left over in the compute ephemeris method.

Reply:
This was mine from a while ago. We can pull the comment.